purefa/manifest.json

```diff
@@ -0,0 +1,55 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a2d8f393-62cd-4ece-bfab-e30797698b12",
+  "app_id": "purefa",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+    "Category::OS System",
+    "Category::Data Store",
+    "Offering::Integration"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor the performance and utilization of Pure Storage FlashArrays",
+    "title": "Pure Storage FlashArray",
+    "media": [
+      {
+        "caption": "Pure Storage FlashArray - Overview",
+        "image_url": "images/FA-overview.png",
+        "media_type": "image"
+      }
+    ]
+  },
+  "author": {
```

Comment:
There are a few more fields required for the author object, here's an example - 
```
  "author": {
        "homepage": "https://www.acme.corp",
        "name": "ACME",
        "support_email": "support@acme.corp",
  }
```

Reply:
Ack